packages/commonwealth/server/migrations/20240708201453-add-profile-to-users.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
+    await queryInterface.sequelize.transaction(async (t) => {
+      await queryInterface.sequelize.query(
+        `
+        ALTER TABLE public."Users" ADD COLUMN "profile" JSONB NOT NULL DEFAULT '{}'::jsonb;
+        WITH FirstProfile AS (
+            SELECT DISTINCT ON (user_id) 
+                  user_id, 
+                  jsonb_build_object(
+                      'name', profile_name,
+                      'email', email,
+                      'website', website,
+                      'bio', bio,
+                      'avatar_url', avatar_url,
+                      'slug', slug,
+                      'socials', socials,
+                      'background_image', background_image
+                  ) AS profile_json
+            FROM "Profiles"
+            ORDER BY user_id, created_at
+        )
+        UPDATE "Users"
+        SET profile = COALESCE(FirstProfile.profile_json, '{}'::jsonb)
+        FROM FirstProfile
+        WHERE "Users".id = FirstProfile.user_id;
+        CREATE OR REPLACE FUNCTION update_user_profile() RETURNS TRIGGER AS $$
```

Comment:
Why do we have a trigger to update the user? Shouldn't it be done in application code for the user creation/updating? IMO having a trigger hides the logic.

Reply:
This is just a temporary fix to keep the old and new models in sync... once we finish the bucket, the old model and trigger will be deleted